packages/jest-message-util/src/index.ts

```diff
@@ -390,9 +389,11 @@ export const separateMessageFromStack = (
   // (maybe it's a code frame instead), just the first non-empty line.
   // If the error is a plain "Error:" instead of a SyntaxError or TypeError we
   // remove the prefix from the message because it is generally not useful.
-  const messageMatch = content.match(
-    /^(?:Error: )?([\s\S]*?(?=\n\s*at\s.*:\d*:\d*)|\s*.*)([\s\S]*)$/,
-  );
+  const ERROR_REGEXP = /^(?:Error: )?([\s\S]*?(?=\n\s*at\s.*:\d*:\d*)|\s*.*)([\s\S]*)$/;
+  const messageMatch =
+    typeof content !== 'string'
+      ? `${content}`.match(ERROR_REGEXP)
```

Comment:
not sure it makes sense to `toString` it. Can we fix `formatExecError` instead? I think that's where the issue comes from rather than `separateMessageFromStack`

Reply:
Yes, you're right.